src/viewPanel/gridStackItem.ts

```diff
@@ -1,12 +1,14 @@
 import { Panel, Widget } from '@lumino/widgets';
 import { Toolbar, ToolbarButton, closeIcon } from '@jupyterlab/ui-components';
+import { ISignal, Signal } from '@lumino/signaling';
 
 export class GridStackItem extends Panel {
   constructor(options: GridStackItem.IOptions) {
     super();
     this.removeClass('lm-Widget');
     this.removeClass('p-Widget');
     this.addClass('grid-stack-item');
+    this.addClass('glue-item');
 
     const { cellIdentity, cell, itemTitle = '', pos, size } = options;
```

Comment:
Should we add this `cellIdentity` into an HTML attribute of this item? If so, in the `_evtContextMenu` handler, we can get the `cellIdentity` of the selected item immediately without doing the filtering logic in `_findItem`.

Reply:
yes, if the target is a `GridStackItem` node and you set the `cellIdentity` attribute of this node before.